soul-admin/src/test/java/org/dromara/soul/admin/service/SelectorServiceTest.java

```diff
@@ -277,7 +277,9 @@ private SelectorDTO buildSelectorDTO(final String id) {
         selectorConditionDTO1.setId("111");
         SelectorConditionDTO selectorConditionDTO2 = new SelectorConditionDTO();
         selectorConditionDTO2.setId("222");
-        selectorDTO.setSelectorConditions(Arrays.asList(selectorConditionDTO1, selectorConditionDTO2));
+        SelectorConditionDTO selectorConditionDTO3 = new SelectorConditionDTO();
+        selectorConditionDTO2.setId("");
+        selectorDTO.setSelectorConditions(Arrays.asList(selectorConditionDTO1, selectorConditionDTO2, selectorConditionDTO3));
```

Comment:
What this for?

Reply:
sorry. it's should be selectorConditionDTO3.